tutorials/optical_tactile_plugin.md

```diff
@@ -53,19 +48,25 @@ The following assumptions have been made when working on this plugin:
 
 - Currently, the plugin's model should be made up of one link, one contact
 sensor and one depth camera.
-- The geometry of the sensor must be a box. Ideally, the size should be around
-20x20x5 mm, but this can be changed.
+- The geometry of the sensor must be a box, because the interpolation of the 
+contacts returned assumes 4 points of input. The visualization of contacts 
+and the `<extended_sensing>` parameter currently make this assumption as well.
+Ideally, the size should be around 20x20x5 mm, but this can be changed. That is
+the approximate size of a real tactile sensor. A bigger one will also work,
```

Comment:
Contact sensors really vary too much to have one consensus approximate size.
```suggestion
Ideally, the size should be around 20x20x5 mm, but this can be changed. A bigger one will also work,
```

Reply:
457691b